module/data/spell/Wildjammer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"spell": [
+		{
+			"name": "Hold Helm",
+			"source": "WJMAiS",
+			"effects": [
+				{
+          "changes": [
+            {
+              "key": "flags.midi-qol.OverTime",
+              "mode": 5,
+              "value": "turn=end,\nsaveAbility=wis,\nsaveDC=@attributes.spelldc,\nlabel=Hold Helm",
+              "priority": "20"
+            },
+            {
+              "key": "StatusEffect",
+              "mode": 0,
+              "value": "Convenient Effect: Paralyzed",
+              "priority": "20"
+            }
+          ],
+					"requires": {
+            "midi-qol": true,
+						"dae": true,
+            "dfreds-convenient-effects": true
+					}
+				}
+			]
+		}
+  ]
+}
```

Comment:
Some metadata for non-`__core` files would be useful, so we can track who's made what at the very least. I'm thinking akin to the existing 5eT `_meta` object, at least for
- the source `json` unique identifier (in case any brews have identical names)
- `convertedBy` (so we know who to ping when there's a problem; need not match the 5eT brew array)
- `version` (could have the patcher log a UI warning if the 5eT brew and patcher files don't have matching versions)

This is on the Agenda™ in #plutonium-addon-data if you want to discuss it more freely than here btw

Reply:
Ah, yes, I didn't make this clear: the filenames here need to 1:1 match the JSON source name in the brew repo for the thing to function. **This file is therefore broken,** as it should be `WJMAiS.json`, if that is indeed the JSON source name.

JSON source name uniqueness is enforced over in the brew repo, so there should never be any collisions. If some clown has used non-filename-safe characters in their source name, then those can just be swapped out, because the JSON source is never displayed anywhere in the UI. (Might even slap a `[a-zA-Z0-9]+` test on the brew repo to enforce it, too!).

As for the rest, my assumption was: "go look up the appropriate file in the brew repo." It's a bad one, though, since `convertedBy` for e.g. the effects and for the brew might be completely different, so agree that a `_meta` is warranted here too. 

Version matching would make sense; good shout.

Reply:
Filenames matching the JSON `source` makes sense. 👌

I can also guarantee that there's at least hyphens and underscores in JSON `source` names, since I've used them, though they are filename-safe... It's probably a big mess, time to rope Lyra into cleaning it up again? 😏

Reply:
@surged20 see #4 or current master branch for the `_meta` format

Reply:
> I'll rebase against master with that and the filename update.

👍 perfect, thank you

> Given the merge/overwrite comments above, should I switch to a merge strategy for the data?

I vote leave it as-is (which is effectively "overwrite") for now. I don't like the data structure I blitzed out for tagging things as "merge," so would want to change that first regardless.